typescript/web/src/utils/tests/data.fixtures.ts

```diff
@@ -31,6 +47,46 @@ export const BASIC_DATASET_DATA: DatasetData = {
   slug: "my-test-dataset",
   workspace: BASIC_WORKSPACE_DATA,
   labelClasses: [],
+  images: [
+    {
```

Comment:
In #841 @Jordanlelay created some data which you could reuse (and eventually make it use your URLS which are closer to the reality).

Reply:
why not the other way around? wouldn't it make more sense that `use-image-navigation.fixtures.ts` imports all from `data.fixtures.ts`, since it's already importing `BASIC_DATASET_DATA`?

Reply:
Indeed it has to be placed at the lowest level, yes 😉

Reply:
You've marked the comment as resolved by yourself but I couldn't validate the changes before and I see no constant like `IMAGE_1_DATA` being reused here.

Reply:
I closed it because the situation shifted, since you seemed to agree to my proposition of basing on `data.fixtures.ts` instead of `use-image-navigation.fixtures.ts` as the mock data source for images.

So I assumed you would make a comment on the changes I made to `use-image-navigation.fixtures.ts` instead.